### PR TITLE
Add Openn311_ServiceRequest.

### PR DESCRIPTION
**Pull request by TakashiAbe0309**
I wrote a gas to register the data of Open311_ServiceRequest in Orion.

### CODE REVIEW

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+function onEditIssues() {
+  var sheet = SpreadsheetApp.getActive().getSheetByName('Issues');
```
Comment (YujiAzama) [inline Open311_ServiceRequest/src/open311_ServiceRequest.gs]: I think the sheet name should define the type of standard data model. (e.g. Open311ServiceRequest)

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  var lastRow = sheet.getLastRow();
+  var idName = "service-request" + lastRow;
+  sheet.getRange(lastRow, 1).setValue(idName);
+  var id = sheet.getRange(lastRow, 1).getValue();
+  var description = sheet.getRange(lastRow, 2).getValue();
+  var location = sheet.getRange(lastRow, 3).getValue();
+  var splitRow = location.indexOf(',');
+  var latitude = location.slice(0, splitRow);
+  var longitude = location.slice(splitRow + 1)
+  var media_url = sheet.getRange(lastRow, 4).getValue();
```
Comment (YujiAzama) [inline Open311_ServiceRequest/src/open311_ServiceRequest.gs]: You should use lowerCamelCase for local variables.

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  var service_code = sheet.getRange(lastRow, 5);
```
Comment (YujiAzama) [inline Open311_ServiceRequest/src/open311_ServiceRequest.gs]: ditto

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  var service_name = sheet.getRange(lastRow, 6).getValue();
```
Comment (YujiAzama) [inline Open311_ServiceRequest/src/open311_ServiceRequest.gs]: ditto

**Review by YujiAzama — CHANGES_REQUESTED** [review]

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs (hunk 1 above)
Comment (TakashiAbe0309) [reply Open311_ServiceRequest/src/open311_ServiceRequest.gs]: Thank you for the review.
I change the spreadsheet name from Issues to Open311ServiceRequest.

File: Open311_ServiceRequest/src/open311_ServiceRequest.gs (hunk 2 above)
Comment (TakashiAbe0309) [reply Open311_ServiceRequest/src/open311_ServiceRequest.gs]: Thank you for the review. I fix the pointed-out part.

**Review by TakashiAbe0309 — COMMENTED** [review]
I fixed what was pointed out in the review.
I changed the sheet name of the script from Issues to Open311ServiceRequest.
I modified the local variable of the script to lowerCamelCase.

**Review by YujiAzama — APPROVED** [review]
LGTM

**Pull request merged**